force-app/main/default/aura/CandidateGrade/CandidateGrade.cmp

```diff
@@ -4,6 +4,11 @@
     <aura:attribute name="displayGrade" type="Boolean" default="false" />
     <aura:handler name="init" value="{!this}" action="{!c.doInit}"/>
 
+    <ltng:require
```

Comment:
### Testing Best Practice: Untested method
#### Severity
Warning

#### Finding
Method `execute` is not tested. Please consider adding a test method to ensure it works as expected.

#### Why is this a problem?
Unit tests ensure that all code meets quality standards before it's deployed. Testing all your code is essential not only to meet Salesforce test coverage requirements but, more importantly, to make sure your logic keeps working as expected as your application evolves.

#### Help and documentation

* [Apex Testing (Trailhead)](https://trailhead.salesforce.com/content/learn/modules/apex\_testing) 
* [Testing Apex (Developer Documentation)](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex\_testing.htm)